packages/scripts/config/rollup.config.base.js

```diff
@@ -4,14 +4,14 @@ const commonjs = require('@rollup/plugin-commonjs');
 const json = require('@rollup/plugin-json');
 const postCss = require('rollup-plugin-postcss');
 const replace = require('@rollup/plugin-replace');
-const { terser } = require('rollup-plugin-terser');
-const typescript = require('@rollup/plugin-typescript');
 const url = require('@rollup/plugin-url');
 const svgr = require('@svgr/rollup').default;
 const autoPrefixer = require('autoprefixer');
 const postCssColorFix = require('postcss-colorfix');
 const postCssNested = require('postcss-nested');
 const postCssUrl = require('postcss-url');
+const esbuild = require('rollup-plugin-esbuild');
+const { terser } = require('rollup-plugin-terser');
```

Comment:
`rollup-plugin-esbuild` claims to also replace `rollup-plugin-terser` (for prod builds) but I don't know if it's actually faster

Reply:
I ran a quick test and the total runtime is comparable (with `rollup-plugin-terser` running slightly faster by a second). There's some custom logic in our rollup terser plugin for preserving comments which I didn't want to change. I'll investigate esbuild's minify functionality more broadly when experimenting with it for end-to-end builds.